app/services/foreman_templates/template_importer.rb

```diff
@@ -25,6 +25,21 @@ def initialize(args = {})
     end
 
     def import!
+      if @repo.start_with?('http://', 'https://', 'git://')
+        import_from_git
+      elsif repo.start_with?('/')
```

Comment:
Hm. The only concern is number of irc questions :-) thinking about this more, also paths like ~foreman/ or ~jenkins/ would be useful. Maybe we can keep filesystem as `else` and try to validate paths later if it seems necessary. Documentation is the key though. After all these changes, we need to write the manual.